.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,61 +1,51 @@
 ---
 name: 🐛 Bug report
 about: Create a report to help us improve
-
 ---
+## Bug report
+<!--- Verify first that your issue is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
```

Comment:
Does this line belong in the `Ansible version` section?

Reply:
Good question. The reason we put it up front is to ensure the user did this *prior* to opening the bug report. Obviously we failed, as he did open it at this point ;-) But it's better to make him check first, rather than having him explain it first, then see for himself that the bug is already fixed.